### ISSUE TITLE
Disaaperance of one download button in case of images and videos downloads 

### ISSUE THREAD

**Issue by Envy49** [Neet-Nestor/Telegram-Media-Downloader#92]
When i open telegram brower * tested in both the versions of it , i use to get two download buttons when i used to download an image but 1 download button acting as a function where i can select the location of it and place it there on my computer and that first button is gone now and i no longer see it when its the case of photos and videos but in videos i can locate it at the bottom right when the video is playing , but in case of pictures thats not the case , i used to like the design before , where we had two download buttons 1st one being the download will be directed towards the location you chose and the second one is the quick download option, please fix it , or provide some advice.

**Comment by GustWind-byte** [comment]
我今天也遇到了相同的问题

**Comment by Envy49** [comment]
Yeah , i hope there is a fix soon..

**Comment by Neet-Nestor** [comment]
I tested it just now but both buttons are visible to me and work fine.

Could you provide the script version and any related console output?

**Comment by Babushkarubaba** [comment]
I have the exact same problem, I tried with all the telegram versions you provided. I absolutely need to get a video back before tonight... Please.

**Comment by Babushkarubaba** [comment]
I found it! You have to put the extension in developer mode and update

**Comment by Neet-Nestor** [comment]
Yes, as stated in the installation instruction, you have to enable developer mode in Chrome for Tampermonkey to work.

> (Note, If you are using Tampermonkey extension in a Chrome-based browser, following [instructions](https://www.tampermonkey.net/faq.php#Q209) to enable Developer Mode.)

**Comment by Envy49** [comment]
script version 
1.202
<img width="682" alt="Screenshot 2024-10-15 at 11 41 14 PM" src="https://github.com/user-attachments/assets/acad9e69-7a4f-4195-a3ae-5053a1d6abf7">
This is the error i got 
tamper monkey ver 5.3.1 
Version 129.0.6668.91 (chrome version)

<img width="1438" alt="Screenshot 2024-10-15 at 11 49 32 PM" src="https://github.com/user-attachments/assets/c0ce6099-3d1f-4771-bc5c-92c3db1e4b8f">

**Comment by Envy49** [comment]
> I tested it just now but both buttons are visible to me and work fine.
> Hey did u try it in a private channel where saving is restricted cause where the downloads arent restricted its showing me two downloads as usual but the problem occurs when im in a private channel where saving is disabled , then one of the download arrow dissapers.

**Comment by Envy49** [comment]
> I found it! You have to put the extension in developer mode and update

Heyy i already had it in developer mode , but still i am only getting a single download button...

**Comment by Neet-Nestor** [comment]
> Heyy i already had it in developer mode , but still i am only getting a single download button...

Hmmm there is nothing printed in the console which is weird. Very likely the script is not running at all.

@Envy49  Could you try updating to v1.203 in which I added more logging and share the console on telegram webapp again?

**Comment by Envy49** [comment]
Heyy , sure i will update the script and keep u updated.
